content/blobstore-tls.md

```diff
@@ -0,0 +1,83 @@
+!!! note
+    Applicable for director version 266.2.0+
+
+    Linux stemcells v3586.5+ (agent 2.83.0)
```

Comment:
For reference, wondering what we relied on to specify this stemcell version ?

Reply:
For linux stemcells I looked at the agent version that first had the ability to use the blobstore over TLS which was 2.83.0: 
- bosh-davcli took on the modern config in https://github.com/cloudfoundry/bosh-davcli/commit/aee928c4e77e19901f525b6e75c7307eaf52accd starting in 0.0.26
- bosh-davcli 0.0.28 (0.0.26 included) was included in agent starting at 2.83.0 in a bump. https://github.com/cloudfoundry/bosh-agent/commit/c0812113d7ecf9c0df8555b346cef15bf2a1e435